### PR TITLE
docs: update guidlines for reporting security issues

### PR DESCRIPTION
**Pull request by raisedadead**
This update adds some quick notes about "beg bounties" and restructures the files a bit for use with both the contribute site and GitHub. We can deliberate on adding/removing things for example about guidelines about other platforms in a separate PR.

### CODE REVIEW

File: docs/security.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ This document outlines our security policy for the codebase, and how to report v
```
Comment (nhcarrigan) [inline docs/security.md:14]: I could be wrong, but doesn't moving this file break the GitHub UI for security policy?

Comment (raisedadead) [reply docs/security.md:14]: They say, it won't: https://docs.github.com/en/code-security/getting-started/adding-a-security-policy-to-your-repository#about-security-policies

**Comment by nhcarrigan** [comment]
Okay one last question - do we want any of these to appear in the docs navbar?

Or in the FAQ:
image

**Review by ojeytonwilliams — APPROVED** [review]
The language LGTM :+1:

**Comment by raisedadead** [comment]
> Okay one last question - do we want any of these to appear in the docs navbar?

Probably yes. But I am about update the "autoreply" to the email with the copy, so we should be fine. I didn't add the doc on the navbar, intentionally because the doc itself could use some updates based on the multiple platforms we had. 

I can add it if you recommend we do that now.

**Comment by nhcarrigan** [comment]
We can look at that in a future iteration, since it needs an overall restructure.

**Review by naomi-lgbt — APPROVED** [review]

**Pull request merged**